### DISCUSSION TITLE
Issue with Theme not Rendering Local Images

### DISCUSSION THREAD

**Discussion by shixiaocaia** [colinwilson/lotusdocs#55] [Q&A] [answered]
Thanks for your theme. I really like it.

However, I'm facing an issue with a blog theme's code where it doesn't seem to properly render local images. I've tried various approaches, but I can't seem to get it to work as expected.

It renders successfully like this when I use an external link:

```shell
![New Lotus Docs Site - Example Content](https://res.cloudinary.com/lotuslabs/image/upload/v1690992310/Lotus%20Docs/images/lotus_docs_new_site_and_content_module_setup_oiuyex.png)
```

But I want to save pictures locally like this, and it fails to render when I run `hugo server`:

```shell
![20230907102715106](../images/20230907102715106.png)
```

Any help or guidance on resolving this issue would be greatly appreciated. Thank you in advance for your assistance!

**Comment by colinwilson** [comment, marked as answer]
> Thanks for your theme. I really like it.

Thanks 🙏

If your image is located in:
```treeview
assets/
└── images/
    └── 20230907102715106.png
```

Try:
```
![20230907102715106](images/20230907102715106.png)
```

**Reply by shixiaocaia** [reply]
thanks!
